src/RAWcookedBundle/Resources/views/Default/index.html.twig

```diff
@@ -0,0 +1,290 @@
+{% extends 'RAWcookedBundle::base.html.twig' %}
+
+{% block title %}RAWcooked{% endblock %}
+
+{% block body %}
+<section>
+    <div class="row reorder-xs">
+        <div class="v-align carousel-padding-xs col-xs-12 col-sm-5">
+            <div>
+            {% image '@RAWcookedBundle/Resources/public/img/RAWcooked_256.png' %}
+                <img src="{{ asset_url }}" alt="RAWcooked logo" class="img-responsive center-xs" title="RAWcooked logo" />

```

Comment:
Is this HTML5? Then `<img src="{{ asset_url }}" […] title="RAWcooked logo">` without ` /`.